Assets/Decontainment/Singletons/Scripts/PromptSystem.cs

```diff
@@ -0,0 +1,63 @@
+﻿using UnityEngine;
+using System.Collections;
+using TMPro;
+using UnityEngine.UI;
+
+public class PromptSystem : PersistentSingleton<PromptSystem>
+{
+    private GameObject obj;
+    private TextMeshProUGUI text;
+    private Image image;
+    private Banner banner;
+
+
+    void Awake()
+    {
+        banner = FindObjectOfType<Banner>();
+        obj = banner.gameObject;
```

Comment:
You don't need obj here. Just use banner.GetComponent when getting component references.